pages/about.html

```diff
@@ -0,0 +1,249 @@
+          </ul>
+        </div>
+        <div class="title-container">
+          <p class="hello">"World-Health-Organization"</p>
+          <h1 class="title">
+            Seventy-fifth World Health <br />
+            Assembly <br />
+            2022
+          </h1>
+        </div>
+
+        <p class="supporting-text">
+          The World Health Assembly is the decision-making body of WHO. It is
+          attended by delegations from all WHO Member States and focuses on a
+          specific health agenda prepared by the Executive Board.
+        </p>
+        <div class="contact">
+          <p>
+            Please contact us per Email for any further questions about WHO
+            2022!
+          </p>
+          <h3>alusinejalloh800@gmail.com</h3>
+        </div>
+      </section>
+
+      <section class="about-logo-section">
+        <div class="title-group">
+          <h2 class="title">WORLD-HEALTH-ORGANIZATION 2022 Logo</h2>
+          <hr />
+        </div>
+
+        <p class="supporting-text">
+          Seventy-fifth World Health Assembly. You can refer contact @
+          alusinejalloh800@gmail.com
+        </p>
+
+        <div class="logo-container">
+          <div class="logo">
+            <img src="../images/h-logo-white.svg" alt="logo" />
+          </div>
+          <small>2022.08.09 (MON)~ 04 (FRI)</small>
+        </div>
+      </section>
+
+      <section class="past-events-section">
+        <div class="title-group">
+          <h2 class="title">See past WHO events</h2>
+          <hr />
+          <p class="supporting-text">
+            The two most famous World Health Organization Assembly tournaments
+            in history
+          </p>
+        </div>
+        <ul class="event-list">
+          <li class="event event1">
+            <div class="event-info">
+              <h2 class="date">18-22 April 1977</h2>
+              <p>30th World Health Assembly, Geneva</p>
+            </div>
+          </li>
+
+          <li class="event event2">
+            <div class="event-info">
+              <h2 class="date">20–22 May 2019</h2>
+              <p>72nd Word Health Assembly in Geneva</p>
+            </div>
+          </li>
+        </ul>
+      </section>
+
+      <section class="partners-section">
+        <div class="title-group">
+          <h2 class="title">Partners</h2>
+          <hr />
+        </div>
+        <ul>
+          <li>HEPA Europe</li>
+          <li>ASEAN</li>
+          <li>NITAGs</li>
+          <li>SEEHN</li>
+          <li>ETAGE</li>
+        </ul>
+      </section>
+    </main>
+
+    <footer class="license">
+      <a class="logo" href="#">
+        <img src="../images/h-logo-white.svg" alt="logo" />
+      </a>
+      <p class="logo-text">
+        2022 World health Assembly. All Rights Reserved. This is registred under
+        the MIT license. for more info, please
+        <a href="https://github.com/Alusp/World-Health-Assembly">See here</a>
+      </p>
+    </footer>
```

Comment:
- [ ] Lİke the last code reviewer said kindly change the background color of the footer on the about and index page. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=You%20should%20follow,space%20between%20elements.)